src/drive/mobile/store/configureStore.dev.js

```diff
@@ -0,0 +1,45 @@
+/* global __DEVELOPMENT__ */
+import { saveState } from './persistedState'
+import RavenMiddleWare from 'redux-raven-middleware'
+import { createLogger } from 'redux-logger'
+import { ANALYTICS_URL, getReporterConfiguration } from '../lib/reporter'
+import { compose, createStore, applyMiddleware } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+import rootReducer from 'drive/mobile/reducers'
+
+const configureStore = (initialState = {}) => {
+  const loggerMiddleware = createLogger()
+  const ravenMiddleWare = RavenMiddleWare(
+    ANALYTICS_URL,
+    getReporterConfiguration()
+  )
+  const middlewares = [thunkMiddleware, ravenMiddleWare]
+  middlewares.push(loggerMiddleware)
+
+  // Enable Redux dev tools
+  const composeEnhancers =
+    (__DEVELOPMENT__ && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
```

Comment:
Testing `__DEVELOPMENT__ ` is unnecessary here since the file is only loaded when `__DEVELOPMENT__ ` is truthy :)